starport/cmd/scaffold_module.go

```diff
@@ -43,23 +43,23 @@ ibcRouter.AddRoute(ibctransfertypes.ModuleName, transferModule)
 app.IBCKeeper.SetRouter(ibcRouter)`),
 )
 
-// NewModuleCreate creates a new module create command to scaffold an
+// NewScaffoldModule  creates a new module create command to scaffold an
```

Comment:
```suggestion
// NewScaffoldModule returns the command to scaffold an
```

I think this is the format we should have to document the command constructor methods